floodsub.go

```diff
@@ -126,12 +132,16 @@ func (p *PubSub) processLoop(ctx context.Context) {
 			}
 		case treq := <-p.getTopics:
 			var out []string
-			for t := range p.myTopics {
-				out = append(out, t)
+			for t, subs := range p.myTopics {
+				if len(subs) > 0 {
```

Comment:
I just searched for occurences of myTopics and it seems there is no way that it's empty. I'll just drop the check. Thanks!